### PR TITLE
Release 2.0.2

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Release 2.0.2
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Release Test

### CODE REVIEW

**Review by atomiix — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by ghost** [comment]
https://github.com/PrestaShop/statscatalog/pull/18

Before QA

**Comment by sarahdib** [comment]
@Progi1984  this pr is ready for QA ? review is required.

**Review by PierreRambaud — DISMISSED** [review]

**Review by sowbiba — DISMISSED** [review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+GitHub contributors:
+--------------------------------
+ - Damien Metzger
+ - Francois Gaillard
+ - Jerome Nadaud
+ - Jérôme Nadaud
```
Comment (atomiix) [inline CONTRIBUTORS.md]:
```suggestion
 - Jérôme Nadaud
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @Progi1984

**Pull request merged**